osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/infra/jms/protocol/outbound/ProtocolResponseMessageSender.java

```diff
@@ -32,21 +34,42 @@ public void send(
       final ProtocolInfo protocolInfo,
       final MessageMetadata messageMetadata) {
 
+    this.sendWithDestination(responseMessage, messageType, protocolInfo, messageMetadata, null);
```

Comment:
Maybe use the jmsTemplate.getDefaultDestination() here, instead of using null as destination. Now you add null, and you have to deal with this null further down the road.

Reply:
I agree, that's a better way to do it.